app/helpers/petition_helper.rb

```diff
@@ -0,0 +1,32 @@
+module PetitionHelper
+  def render_petition_form(petition, form)
+    [
+      render_hidden_details(petition, form),
+      render_ui(petition, form)
+    ].join.html_safe
```

Comment:
Could probably use [`safe_join`][1] here - though can't we just use `capture` and `concat`?

[1]: http://api.rubyonrails.org/classes/ActionView/Helpers/OutputSafetyHelper.html#method-i-safe_join